manifests/params.pp

```diff
@@ -1,10 +1,63 @@
+# This class takes a small number of arguments (can be set through Hiera) and
+# generates sane default values installation media names and locations.
+# This is a parameters class, and contributes no resources to the graph.
+# Rather, it only sets values for parameters to be consumed by child classes.
 class nessus::params {
 
-  $package_name    = 'Nessus'
```

Comment:
IIRC, the rpm package is named specifically with an uppercase N as the package name.  This should be simple enough to handle as a selector for each OS.